react/features/conference/middleware.js

```diff
@@ -35,6 +39,12 @@ MiddlewareRegistry.register(store => next => action => {
 
         break;
     }
+    case KICKED_OUT: {
+        store.dispatch(
+            conferenceFailed(action.conference, JitsiConferenceEvents.KICKED));
+        store.dispatch(appNavigate(undefined));
```

Comment:
These are really minor styling nitpicks which I'm happy to apply myself (because, additionally, they're probably questions of personal preference in a way):
* I usually tend to leave a blank line between `case` blocks to signify their logical separateness.
* In order to make lines shorter, when I have to write multiple `store.dispatch` (or `store.getState`) in a block, I tend to shorten the lines for readability by decomposing store beforehand like `const { dispatch } = store;`